README.md

```diff
@@ -104,27 +104,40 @@ The following needs to be done just once per setup.
     ```
 
 ### Linking
-1. In your application directory: Link the required OpenUI5 libraries
-    ```sh
-    npm link @openui5/sap.f
-    npm link @openui5/sap.m
-    npm link @openui5/sap.ui.core
-    npm link @openui5/themelib_sap_fiori_3
+
+Would you like to work on the application project and one or more of its UI5 framework dependencies at the same time? We got you covered!
+
+1. Create a new file `ui5-workspaces.yaml` in the root folder of the project, right next to the `ui5.yaml`
+2. In `ui5-workspace.yaml` add the paths to the local dependencies you'd like to use from the local machine:
+    ```yaml
+    specVersion: workspace/1.0
+    metadata:
+        name: default
+    dependencyManagement:
+        resolutions:
+            # local path to OpenUI5. It'd resolve all required libraries and transitive dependencies.
```

Comment:
It'd = ?

Reply:
https://dictionary.cambridge.org/dictionary/english/it-d

I will change it